### ISSUE TITLE
Rule: Concept should include core props only

### ISSUE THREAD

**Issue by nickevansuk** [openactive/data-model-validator#150]
* **Target Model(s):** `Concept`
* **Target Field(s):** `definition`, `broader`, `narrower`, `related`
* **Severity:** Warning
* **Category:** Data Quality

`"definition"`, `"broader"`, `"narrower"` and `"related"` should not be included as part of the Concept, as these are referenced from the source list

Note it should also recognise the `skos:` prefix here if people are just copying from the examples (see https://github.com/openactive/data-model-validator/issues/149). 

Not like this:

```javascript
"activity": [ {
  "id": "http://openactive.io/activity-list/#18383130-20e1-4a19-980b-ef9c518c3b72",
  "type": "skos:Concept",
  "prefLabel": "Fitness League",
  "skos:definition": "Fitness League is used to deliver 'Bagot Stack' which is designed to improve posture and well-being through core stability and rhythmic exercise to music.",
  "broader": "http://openactive.io/activity-list/#d3b5104a-2e31-4cca-a278-ef8e0987a764"
} ],
```

Like this:

```javascript
"activity": [
  {
    "id": "http://openactive.io/activity-list/#18383130-20e1-4a19-980b-ef9c518c3b72",
    "type": "Concept",
    "prefLabel": "Fitness League"
  }
],
```

**Comment by MelanieAbraham** [comment]
@ldodds is this in the spec?

**Comment by ldodds** [comment]
@nickevansuk has left some other comments on the validation rules doc which are related. There's several contexts in which a Concept might appear:

* "in-line" in a feed or API response
* as part of an Activity List

At the moment the validator doesn't really distinguish between these, but we may want it to do that in future. The rules for what could or should appear will be slightly different.

The fields Nick highlights shouldn't be necessary in a feed or should be included when validating a list. I'm happy to include a warning for those items, but ONLY if the Concept has an inScheme property. If it doesn't then including them is fine (but not ideal).

Basically: if the Concept is from a standard list, warn people that things like definition, broader, narrower, related aren't necessary and should be in the list. Otherwise ignore.
